plugins/tiddlywiki/codemirror/engine.js

```diff
@@ -91,6 +91,7 @@ function CodeMirrorEngine(options) {
 	this.value = options.value;
 	this.parentNode = options.parentNode;
 	this.nextSibling = options.nextSibling;
+	this.editorManager = $tw.editorManager[self.widget.editorStateQualifier];
```

Comment:
I didn't review this file.

Reply:
here the manager gets actually used - init problems beside